### PR TITLE
Update WordPress plugin docs for WP v4

### PR DESCRIPTION
**Pull request by joshcanhelp**
Update documentation to align with the [v4.0.0 release](https://github.com/auth0/wp-auth0/releases/tag/4.0.0).

### CODE REVIEW

File: articles/cms/wordpress/configuration.md [hunk 1]
```diff
@@ -11,34 +11,28 @@ useCase:
...
+  - **If you're troubleshooting the Setup Wizard**, navigate to the [Applications](${manage_url}/#/applications) screen and look for an Application that is similar to your WordPress site name. If you don't find one, it means that an Application was not created by the Wizard. Restart the Setup Wizard in WordPress or follow the step just below to create an Application manually.
+  - **If you're configuring manually**, navigate to the [Applications](${manage_url}/#/applications) screen and click **Create Application**. Enter a name for the Application, select **Regular Web Applications**, and click **Create**.
 
     ![Auth0 Applications in the Management Dashboard](/media/articles/cms/wordpress/application-listing.png)
 
-2. Click on the name to get to the **Settings** tab. You will see your Domain, Client ID, and Client Secret, which are used in **wp-admin > Auth0 > Settings** to make a connection to Auth0.
+2. Click on the on the **Settings** tab for the Application. You will see your Domain, Client ID, and Client Secret, which are used in **wp-admin > Auth0 > Settings** to connect to Auth0.
```
Comment (damieng) [inline articles/cms/wordpress/configuration.md]:
```suggestion
2. Click on the **Settings** tab for the Application. You will see your Domain, Client ID, and Client Secret, which are used in **wp-admin > Auth0 > Settings** to connect to Auth0.
```

File: articles/cms/wordpress/configuration.md [hunk 2]
```diff
@@ -47,34 +41,34 @@ First, we'll check for the Application created for your WordPress site.
 4. Scroll down to **Allowed Callback URLs** and provide the WordPress site URL with `?auth0=1` appended:
 
 ```
-https://yourdomain.com/index.php?auth0=1
+https://your-wordpress-domain.com/index.php?auth0=1
 ```
 
 ::: warning
-Do **not** cache Callback URLs, or you might see an "Invalid state" error during login. Please see our [troubleshooting steps for this error](/cms/wordpress/invalid-state#cached-callback-urls) for more information.
+The Callback URL here **must not** be cached, or you might see an "Invalid state" error during login. Please see our [troubleshooting steps for this error](/cms/wordpress/invalid-state#cached-callback-urls) for more information.
```
Comment (damieng) [inline articles/cms/wordpress/configuration.md]:
```suggestion
The Callback URL here **must not** be cached, or you might see an "Invalid state" error during login. Please see [these troubleshooting steps](/cms/wordpress/invalid-state#cached-callback-urls) for more information.
```

File: articles/cms/wordpress/installation.md [hunk 3]
```diff
@@ -42,15 +40,15 @@ The Login by Auth0 plugin is compatible with WordPress multisite networks. The p
 
 There are a few ways that a network of sites can be setup in Auth0:
 
-1. All sites can share both an Application and a database connection:
+1. **All sites can share both an Application and a database connection**
 	1. Run the Setup Wizard steps to completion for the main site.
 	2. Setup all other sites manually using the **Domain**, **Client ID**, and **Client Secret** from the main site in the Basic tab of the Auth0 settings page.
 	3. Update the Application's **Allowed Callback URLs**, **Allowed Web Origins**, and **Allowed Logout URLs** to include each site (wildcards can be used if your network uses subdomains).
-2. Each site can have its own Application and share a database connection:
+1. **Each site can have its own Application and share a database connection (recommended)**
```
Comment (damieng) [inline articles/cms/wordpress/installation.md]: Make the recommended approach the first option in this list?

**Review by damieng — CHANGES_REQUESTED** [review]

File: articles/cms/wordpress/installation.md (hunk 3 above)
Comment (joshcanhelp) [inline articles/cms/wordpress/installation.md]:
```suggestion
1. **Each site can have its own Application and share a database connection**
```

**Review by damieng — APPROVED** [review]

**Pull request merged**